src/components/PlanCard/PlanCard.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import planCard1 from '../../assets/img/plan-card-1.png';
+import {
+  CardContainer,
+  PlanInfoWrapper,
+  PlanInfo,
+  PlanId,
+  PlanDate,
+  PlanNav,
+  OptionIcon,
+  PlanImg,
+} from './PlanCard.styled';
+import { PlanInterface } from '../../types';
+import { getFormattedDate } from '../../utilities/dateFormat';
+
+interface PlanCardProps extends PlanInterface {
+  handleClick(): void;
+  isAdmin?: boolean;
+}
+
+const PlanCard = ({
+  name,
+  startDate,
+  endDate,
+  handleClick,
+  isAdmin = false,
+}: PlanCardProps) => {
+  const formattedStartDate = getFormattedDate(startDate);
+  const formattedEndDate = getFormattedDate(endDate);
+  const planDate = `(${formattedStartDate} - ${formattedEndDate})`;
```

Comment:
can you help delete these please? cause we wont need `startDate` and `endDate` anymore